src/promptflow/promptflow/executor/_process_manager.py

```diff
@@ -368,6 +399,13 @@ def handle_signals(self, control_signal, i):
             self.new_process(i)
 
 
+def create_spawned_fork_process_manager_wrapper(*args, **kwargs):
+    try:
+        create_spawned_fork_process_manager(*args, **kwargs)
+    except Exception as e:
```

Comment:
Do we still need this?

Reply:
Yes, if we remove the wrapper, errors of the incorrect parameters will not generate a corresponding log to prompt the developer, but will hang for 1minutes until get process info fails.

Reply:
remove